src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx

```diff
@@ -148,26 +159,73 @@ const Selector = () => {
 
   const mapWrapper =
     <Wrapper data-testid="containerSelectSelector">
-      <StyledMap hasZoomControls mapOptions={mapOptions} setInstance={setMap} events={{}}>
-        <ViewerContainer
-          topLeft={
-            <ButtonBar>
-              <div>
-                <Button onClick={addContainer}>Container toevoegen</Button>
-                <Button onClick={removeContainer}>Container verwijderen</Button>
-                <Button onClick={close}>Meld deze container/Sluiten</Button>
-              </div>
-              <Paragraph as="h6">
-                Geselecteerd: {selection ? `[${selection.reduce((res, { id }) => `${res},${id}`, '')}]` : '<geen>'}
-              </Paragraph>
-            </ButtonBar>
-          }
-        />
+      <StyledMap
+        data-testid="map"
+        hasZoomControls={showDesktopVariant}
+        mapOptions={mapOptions}
+        setInstance={setMap}
+        events={{}}
+      >
+        <MapPanelProvider
+          mapPanelSnapPositions={{
+            [SnapPoint.Closed]: '90%',
+            [SnapPoint.Halfway]: '50%',
+            [SnapPoint.Full]: '0',
+          }}
+          mapPanelDrawerSnapPositions={{
+            [SnapPoint.Closed]: '30px',
+            [SnapPoint.Halfway]: '300px',
+            [SnapPoint.Full]: '100%',
+          }}
+          variant={panelVariant}
+          initialPosition={SnapPoint.Closed}
+        >
+          <ViewerContainer
+            showDesktopVariant={showDesktopVariant}
+            legendButton={
+              <MapPanelLegendButton
+                showDesktopVariant={showDesktopVariant}
+                currentOverlay={currentOverlay}
+                setCurrentOverlay={setCurrentOverlay}
+              />
+            }
+            bottomRight={
+              <ButtonBar>
+                <div>
+                  <Button size={100} onClick={addContainer}>
+                    Container toevoegen
+                  </Button>
+                  <Button size={100} onClick={removeContainer}>
+                    Container verwijderen
+                  </Button>
+                  <Button size={100} onClick={close}>
+                    Meld deze container/Sluiten
+                  </Button>
+                </div>
+                <Paragraph as="h6">
+                  Geselecteerd: {selection ? `[${selection.reduce((res, { id }) => `${res},${id}`, '')}]` : '<geen>'}
+                </Paragraph>
+              </ButtonBar>
+            }
+          />
+          <Panel data-testid={`panel-${showDesktopVariant ? 'desktop' : 'mobile'}`}>
+            <LegendPanel
+              variant={panelVariant}
+              title="Legenda"
+              items={meta?.featureTypes.map(featureType => ({
```

Comment:
I am getting here a TS2322 error.

Reply:
Can you double check if you're still getting that error?